server/models/Collective.js

```diff
@@ -1862,6 +1862,10 @@ export default function(Sequelize, DataTypes) {
       .then(() => {
         return Promise.map(tiers, tier => {
           if (tier.id) {
+            if (tier.amountType === 'FIXED') {
```

Comment:
Why doing it only if the tier has an id?

Reply:
Good call, doesn't hurt to also do that for creations.